### ISSUE TITLE
is `environment` supposed to be reloaded on -SIGHUP? 

### ISSUE THREAD

**Issue by 01micko** [labwc/labwc#227]
See https://github.com/labwc/labwc-tweaks/pull/3

https://github.com/labwc/labwc-tweaks/pull/3#commitcomment-66006142

**Comment by johanmalm** [comment]
Yes we ought to call `session_environment_init()` again on SIGHUP.
I think we'd have to reload the cursor images too.

**Comment by Consolatis** [comment]
f548fcd31fb142c89e3992cb9ac87e4e6c85bd24 should be enough to reload the envrionment on SIGHUP. It will not change any internal cursor settings though so its only one part of the full solution. Every new application started after the SIGHUP will get the new environment. SIGHUP will also not delete anything from the environment because that would require tracking every key we set (and what the original value was before we set the new one). I am not sure it is worth doing that.

The internal cursor theme is set here:
https://github.com/labwc/labwc/blob/e1328c055dcc1efb191e43421ce7f88378820321/src/cursor.c#L794-L802
I had a quick look at `wlroots/include/wlr/types/wlr_xcursor_manager.h` and there doesn't seem to be anything to change the theme after it has been loaded.

Will investigate a bit more but for now I'll send a PR to just update the environment on SIGHUP.
